lib/chef/knife/vault_show.rb

```diff
@@ -90,9 +90,7 @@ def print_values(vault, item, values)
       def print_keys(vault)
         if bag_is_vault?(vault)
           bag = Chef::DataBag.load(vault)
-          split_vault_keys(bag)[1].each do |item|
-            output item
-          end
+          output split_vault_keys(bag)[1]
```

Comment:
If there's more than 1 key this will only show a single key. Is that intentional?

Reply:
Hi @tas50,yes it is intentional as split_vault_keys method return array of keys and items https://github.com/chef/chef-vault/blob/9e9d28e6dd8c43c32c65d1b69b46ed685a67f504/lib/chef/knife/vault_base.rb#L81.

And regarding the change made in PR, below are some o/ps before change and after change:

**Before Change:**
```
knife vault show val_passwords
   root
   root1
knife vault show val_passwords -F json
   "root"
   "root1"
knife vault show val_passwords -F yaml
	--- root
	...
	--- root1
	...
```

**After Change:**
```
knife vault show val_passwords
	root
	root1
knife vault show val_passwords -F json
	[
  	"root",
  	"root1"
	]
knife vault show val_passwords -F yaml
	---
	- root
	- root1
```